examples/raspberry-pi/rpi-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Define variables for DNS and port
+DNS=
```

Comment:
this should be `ADDRESS` or something more clear, maybe `IP`:

Reply:
> this should be `ADDRESS` or something more clear, maybe `IP`:

if only ipv4 and ipv6 are intended, use _ip_. If both DNSs and IPs, _address_. If any web stuff, _url_,